vms/platformvm/block/executor/proposal_block_test.go

```diff
@@ -199,7 +199,6 @@ func TestBanffProposalBlockTimeVerification(t *testing.T) {
 			}
 			return nil, database.ErrNotFound
 		}).AnyTimes()
-	onParentAccept.EXPECT().Config().Return(env.config, nil).AnyTimes()
```

Comment:
can you please add a comment regarding this removal?

Reply:
Seems that its very old leftover. It wasn't noticed cause of `.AnyTimes() which allows this method to not be called at all.
This PR removes it, cause it also removes `MockState.Config`